webpack.config.js

```diff
@@ -91,6 +94,11 @@ if ( CALYPSO_ENV === 'desktop' || CALYPSO_ENV === 'desktop-mac-app-store' ) {
 } else {
 	webpackConfig.entry.vendor = [ 'react', 'store', 'page', 'wpcom', 'jed', 'debug' ];
 	webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( 'vendor', '[name].[hash].js' ) );
+	webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( {
+		children: true,
```

Comment:
it tells the plugin to look through all of the children of each chunk for common modules. We have a vendor chunk and a main entry chunk, build-$env. All of our other chunks are children of $build-env. So setting `children:true` tells the commons chunk plugin to look through all of those chunks for common modules.